solidity/random-beacon/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@keep-network/random-beacon",
+  "version": "2.0.0-dev",
+  "description": "random beacon v2",
+  "scripts": {
+    "build": "hardhat compile",
+    "test": "hardhat test",
+    "deploy": "hardhat deploy --export export.json"
+  },
+  "dependencies": {
+    "@thesis/solidity-contracts": "github:thesis/solidity-contracts#4985bcf"
+  },
+  "devDependencies": {
+    "@keep-network/hardhat-helpers": "^0.1.0-pre.0",
+    "@keep-network/hardhat-local-networks-config": "^0.1.0-pre.0",
+    "@keep-network/prettier-config-keep": "github:keep-network/prettier-config-keep#a1a333e",
+    "@nomiclabs/hardhat-ethers": "^2.0.2",
+    "@nomiclabs/hardhat-waffle": "^2.0.1",
+    "@types/chai": "^4.2.22",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^16.9.6",
+    "@tenderly/hardhat-tenderly": "^1.0.12",
+    "chai": "^4.3.4",
+    "eslint": "^7.30.0",
```

Comment:
If we want to add eslint, prettier, and solhint dependencies in this PR we should define all the remaining pieces like scripts, workflows, etc. We could also configure linting/formatting in a separate PR.

Basically, let's review all the dependencies here one by one and leave just the ones that actually are needed and used.

Reply:
Yea, initially I wanted to add `eslint`, `prettier`, and `solhint` here in later commits (this PR was a draft). But I think it's a good idea to add it in a separate PR. 2bafc94